operations/installation.rst

```diff
@@ -0,0 +1,43 @@
+.. _installation-operation:
+
+============
+Installation
+============
+
+Omnivector provides a repository to install Slurm from, to make sure all charms
+install a supported Slurm version. The currently supported Slurm version is
+``20.11.8`` on CentOS 7 and ``20.11.7`` on Ubuntu Focal.
+
+Omnivector Slurm Distribution allows one to use a custom repository URL to
+install Slurm from. This is specially useful to allow using a local mirror, or
+cache, of packages.
+
+To specify a custom URL, set the value of ``custom-slurm-repo`` in your
```

Comment:
This sentence is a little bit tricky to parse.  It might be good to find a way to break it up or reword it for a more natural flow.